py/core/providers/chunking/r2r_chunking.py

```diff
@@ -31,11 +31,9 @@ def _initialize_text_splitter(self) -> TextSplitter:
                 chunk_overlap=self.config.chunk_overlap,
             )
         elif self.config.method == Method.BASIC:
-            # Implement basic method
-            pass
+            raise NotImplementedError("Basic method not implemented")
```

Comment:
Nit: this messaging is a bit confusing. We probably want it to be closer to "Basic chunking is not supported. Please use "recursive""